share/extender/build_input.yml

```diff
@@ -251,15 +251,18 @@ platforms:
     env:
         PROGUARD:               "{{env.ANDROID_PROGUARD}}"
         LIBRARYJAR:             "{{env.ANDROID_LIBRARYJAR}}"
-        NDK_PATH:               "{{env.ANDROID_NDK20_PATH}}"
-        SYSROOT:                "{{env.ANDROID_NDK20_SYSROOT}}"
-        NDK_VERSION:            "{{env.ANDROID_NDK20_API_VERSION}}"
-        STL_LIB:                "{{env.ANDROID_NDK20_STL_LIB}}"
+        NDK_PATH:               "{{env.ANDROID_NDK25_PATH}}"
+        PATH:                   "{{env.ANDROID_NDK25_BIN_PATH}}:{{env.PATH}}"
+        SYSROOT:                "{{env.ANDROID_NDK25_SYSROOT}}"
+        NDK_VERSION:            "{{env.ANDROID_NDK25_API_VERSION}}"
         MANIFEST_MERGE_TOOL:    "{{env.MANIFEST_MERGE_TOOL}}"
+        PATH:                   "{{env.ANDROID_NDK25_BIN_PATH}}:{{env.PATH}}"
+        NDK_LD:                 "{{env.ANDROID_NDK25_BIN_PATH}}/ld.lld"
+        NDK_CXX:                "{{env.ANDROID_NDK25_BIN_PATH}}/armv7a-linux-androideabi{{env.ANDROID_NDK25_API_VERSION}}-clang++"
```

Comment:
Taking into account specific aspects of how ProcessBuilder works, it’s important to use absolute path for the compiler. If a relative path is used, then subprocesses of the compiler are looking for the InstalledDir in PATH variable and may find a wrong one (from another SDK). For the absolute path InstalledDir detects from it.